RPI2/Ubuntu/1_create_base_image.md

```diff
@@ -0,0 +1,170 @@
+# RPi Ubuntu setup script for use as companion computer.
+
+These instructions are for the official Ubuntu LTS images. 
+
+Download Ubuntu LTS image at this time 18.04 is the recomended image, for RPi3 and RPi4 use the 64bit Image
+Even though the RPi3 has only a marginal difference between using 32bit vs 64bit, using the 64bit image provides 
+Common ground between our images.
+
+https://ubuntu.com/download/raspberry-pi 
+
+Flash image onto SD card using Balena Etcher which can be downloaded from 
+https://www.balena.io/etcher/
+
+NOTE: make sure you use at least a  16GB card, preferabbly use a 128GB card to allow enough space to install all software.
+
+Boot Raspberry Pi and follow the instructions for changing the password for the default user (User: ubuntu, Password: ubuntu)
+
+Setup is easier if you can ssh into the vehicle from an existing
+desktop environment.  Use the ethernet connection for this.
+
+Once you log in for the first time you can find out the ip-address of the RPi by typing this command on the console (look for the number next to inet):
+```console
+ubuntu@ubuntu:~$ ifconfig
```

Comment:
IIRC, `ifconfig` was not present by default on Ubuntu 18.04, had to use `ip a` to get the info, or install `net-tools`
Don't know if that's the case with the Ubuntu image for Rpi, just mentioning

Reply:
It is definitely present on all distributions I’ve used from the Jetsons to the RPi